server/src/main/java/org/elasticsearch/search/vectors/PatienceCollectorManager.java

```diff
@@ -40,7 +40,7 @@ class PatienceCollectorManager implements KnnCollectorManager {
     }
 
     static KnnCollectorManager wrap(KnnCollectorManager knnCollectorManager, int k) {
-        return new PatienceCollectorManager(knnCollectorManager, Math.max(7, (int) (k * 0.3)), DEFAULT_SATURATION_THRESHOLD);
+        return new PatienceCollectorManager(knnCollectorManager, Math.max(7, (int) (k * 0.1)), DEFAULT_SATURATION_THRESHOLD);
```

Comment:
I've adjusted this to counterbalance both the automatic oversampling (when num_candidates is not set) and the manual num_candidates oversampling (usually set to be > k).